hass-addon-sunsynk-multi/config.yaml

```diff
@@ -58,7 +58,7 @@ schema:
       MODBUS_ID: int(1,16)
       DONGLE_SERIAL_NUMBER: int?
       PORT: str
-  SENSOR_DEFINITIONS: list(single-phase|three-phase)
+  SENSOR_DEFINITIONS: list(single-phase|three-phase|three-phase-hv)
```

Comment:
Can you undo this change for now? Then we can release it on edge and test it for a while

(Changing multi's config without updating the version & tagging it causes inconsistencies between the addon's options & the built container)